### ISSUE TITLE
Feature request: please offer 'On a custom interval' option

### ISSUE THREAD

**Issue by JoSchuh** [james-fray/tab-reloader#52]
Hi, thanks for the great add-on!

I am missing an additional option 'On a custom interval'.
I would love to see it next to the option 'Every hour'.
When clicking it, a little modal dialog could show, offering something with a:
- Text-label: 'Auto refresh page every'
- Three digit numerical input field: [xxx]
- Drop-Down field: ^Seconds|Minutes^
This way, I could easily set 2,3,4 Minutes or 570 Seconds.

The reason for my request: I have two sites that throw me out after 5 or after 10 minutes and I think the preconfigured reload times below these timeouts are too far off :)

I can accept if you say I should use the job system, but I think it would be a handy option...

(Edit: typo)

**Comment by rdhar** [comment]
I think this has since been implemented, right?

**Comment by JoSchuh** [comment]
> I think this has since been implemented, right?

Hi rdhar, when I posted the request, I was solely using TabReloader by rightclicking tabs and accessing the context menu.
Later I placed the TabReloader button to FFs main menu bar to enter time values and use the switch there.
The perfect flavor for me would still be to be able to remove that main menu button and only use rightclicks on tabs.
If I imagine getting the same time control panel from there and have a TabReloader symbol on the tab itself, boy, that would be cool :)
But I'm fine and TabReloader ist GREAT!
